adapter-jdbc/src/main/resources/activities.baselinesv2/cockroachdb-tabular.yaml

```diff
@@ -0,0 +1,137 @@
+min_version: "5.17.1"
+
+description: |
+  A tabular workload with partitions, clusters, and data fields
+  This workload contains partitioning and cluster along with a set
+  of 8 fields of varying length. The field values vary in size according
+  to the fibonacci sequence times a base size factor of 10, with
+  an additional 10% variance for each field.
+  The read patterns have a variety of field subsets specified.
+
+  During rampup, all rows will be written partition by partition,
+  filling in all rows of that partition before moving on to the next.
+  Example: With a partition size of 1000 and 1B rows, there will be
+  1000000 partitions.
+
+  During main phase, the read patterns are varied with different
+  field sets. As well, the number of rows which will be returned
+  is varied between 1 and 10.
+
+  By default, reads occur at the same ratio as writes, with main
+  phase writes writing full rows.
+
+  You can bulk up the size of the payloads by 10x with addzeroes='0',
+  by 100x with addzeroes='00', and so on, but if you want to go higher
+  than 100x, you'll need to modify the workload with a larger reference
+  file in the HashedFileExtractToString(...) bindings.
+
+scenarios:
+  default:
+    schema: run driver=jdbc tags==block:schema threads==1 cycles==UNDEF
+    rampup: run driver=jdbc tags==block:rampup cycles===TEMPLATE(rampup-cycles,10B) threads=auto
+    main: run driver=jdbc tags==block:"main.*" cycles===TEMPLATE(main-cycles,100M) threads=auto
```

Comment:
Change to `tags=block:'main.*'` to emphasize a safer pattern (such as when copy/pasted in shell...)

Reply:
This is now addressed. 👍🏼